operations/iuf/workflows/preparation.md

```diff
@@ -0,0 +1,48 @@
+# Prepare for the install or upgrade
+
+This section defines environment variables and directory content that is used throughout the workflow.
+
+**`NOTE`** The following step uses the `iuf activity` command to demonstrate how to record operations within an IUF activity. While `iuf` automatically records all `iuf run` operations within an IUF activity, any other
+administrative operation can also be recorded within an IUF activity by using `iuf activity` in this manner. The remainder of the workflow will not use `iuf activity`, deferring to the administrator to use it as they desire.
+`iuf activity` does not **need** to be used in this step, but the rest of the operations in the step are required.
+
+- [1. Prepare for the install or upgrade](#1-prepare-for-the-install-or-upgrade)
+- [2. Next steps](#2-next-steps)
+
+## 1. Prepare for the install or upgrade
+
+1. Create timestamped media, activity, and administrator directories on `ncn-m001`. Copy all distribution files from the HPC CSM Software Recipe to the media directory, utilizing `iuf activity` to record the time spent downloading
```

Comment:
Step 1. Create timestamped media...
Do we need a command block for this step?

Step 2 . Copy all distribution files...
Do we need a command block for this step?

Step 3. Record the time spending downloading using `iuf activity`.
Do we need a command block for this step?

Reply:
They are all part of the command block a few lines below the table.  I don't know if there is value in splitting them apart but I can if you feel strongly about it.